psalm-baseline.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.10.0@916b098b008f6de4543892b1e0651c1c3b92cbfa">
```

Comment:
`4.19` is in `composer.lock`, but `4.10.0` is declared here? :thinking:

Reply:
You're right! :)
I'll fix it.